src/marginaldamage.jl

```diff
@@ -39,7 +39,9 @@ function _compute_scc(mm::MarginalModel; year::Int, last_year::Int, prtp::Float6
     ntimesteps = findfirst(isequal(last_year), model_years)     # Will run through the timestep of the specified last_year 
     run(mm, ntimesteps=ntimesteps)
 
-    marginal_damages = -1 * mm[:neteconomy, :C][1:ntimesteps] * 1e12     # Go from trillion$ to $; multiply by -1 so that damages are positive; pulse was in CO2 so we don't need to multiply by 12/44
+    # Go from trillion$ to $; multiply by -1 so that damages are positive; pulse 
+    # was already converted to CO2 so we don't need to multiply by 12/44
```

Comment:
I believe the unit of emissions in DICE2013 was changed to CO2 (from C in DICE2010), so the model just natively runs in CO2, no conversion needed anywhere. Would be good to double check that, though, I might misremember.

Reply:
Yep I think that's right, which is (I believe) what this line does, no inclusion of 12/44 just the 1e12